staff/daniel-toledo/community.html

```diff
@@ -1,249 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Do you have that file empty? Or you deleted the file?

Reply:
I have delated... but it is following me :(